boards/mulle/include/periph_conf.h

```diff
@@ -142,12 +142,23 @@ extern "C"
  */
 static const adc_conf_t adc_config[] = {
     /* dev, pin, channel */
-    { ADC0, GPIO_UNDEF, 26 },       /* internal: temperature sensor */
-    { ADC0, GPIO_UNDEF, 27 },       /* internal: band gap */
-    { ADC0, GPIO_UNDEF, 29 },       /* internal: V_REFSH */
-    { ADC0, GPIO_UNDEF, 30 },       /* internal: V_REFSL */
-    { ADC1, GPIO_UNDEF,  0 },       /* connected to  Mulle Vbat/2 on PGA1_DP */
-    { ADC1, GPIO_UNDEF, 19 },       /* connected to Mulle Vchr/2 on PGA1_DM  */
+    [ 0] = { ADC1, GPIO_UNDEF, 26 },       /* internal: temperature sensor */
```

Comment:
:+1: Good catch. It was intentional with the intention that if you don't use any expansion port analog pins you only need to power the ADC0 module in order to measure battery voltage, charger voltage and, measuring the 1 V band gap to compute the AVDD.